response/response.go

```diff
@@ -102,8 +102,10 @@ func (res *Response) UnmarshalCBOR(data []byte) error {
 	// Please enjoy this: https://github.com/pyfisch/cbor/blob/2f2d0253e2d30e5ba7812cf0b149838b0c95530d/src/ser.rs#L83-L117
 	possiblyString := ""
 
+	fmt.Printf("CBOR data to unmarshal: %v\n", data)
 	err := cbor.Unmarshal(data, &possiblyString)
 	if nil != err {
+		fmt.Printf("Error while unmarshalling: %v\n", err)
```

Comment:
Would you mind removing the printfs here?

```suggestion
	err := cbor.Unmarshal(data, &possiblyString)
	if nil != err {
```

Reply:
Whoops, those should have never been part of the PR. I added the printfs to debug https://github.com/hf/nsm/issues/2. Should be fixed!